debug/genConnectionMetadata.js

```diff
@@ -20,9 +20,22 @@ ${keyData.toString('base64')}
 -----END PRIVATE KEY-----`);
 })();
 
-function genConnectionMetadata() {
+function genConnectionMetadata(options) {
   const now = Date.now();
 
+  const restrictNetwork = !!options?.restrictNetwork;
+  let repl = options?.repl;
+
+  if (repl == null) {
+    repl = {
+      id: `testing-crosis-${Math.random().toString(36).split('.')[1]}`,
+      language: 'bash',
+      slug: Math.random().toString(36).slice(2),
+      user: 'crosistest',
+      bucket: 'test-replit-repls',
+    };
+  }
+
```

Comment:
nit: slightly easier way of writing
```suggestion
  let repl = options?.repl || {
    id: `testing-crosis-${Math.random().toString(36).split('.')[1]}`,
    language: 'bash',
    slug: Math.random().toString(36).slice(2),
    user: 'crosistest',
    bucket: 'test-replit-repls',
  };
```